src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -414,19 +387,20 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 accountIds.add(thisContact.accountId);
             }
 
-            accountsOnContacts = [Select Id, Type, RecordTypeId, RecordType.Name, (Select Id from Opportunities limit 1), (select Id from Contacts limit 1) 
+            accountsOnContacts = [Select Id, Type, RecordTypeId, (Select Id from Opportunities limit 1), (select Id from Contacts limit 1) 
                                   from Account where Id IN :accountIds];
             
             for (Account thisAccount : accountsOnContacts) {
                 //if the legacy or current type field show it's not a HH Account record, don't delete the account
-                if (thisAccount.RecordType.Name == CAO_Constants.HH_ACCOUNT_PROCESSOR || 
-                thisAccount.RecordType.Name == CAO_Constants.ADM_ACCOUNT_PROCESSOR) {
+                //TODO: this should be looking at a setting to determine which types can be deleted (W-009779)
+                if (thisAccount.RecordTypeId == UTIL_Describe.getHhAccRecTypeID() || 
+                thisAccount.RecordTypeId == UTIL_Describe.getAdminAccRecTypeID()) {
                     
                     // if there are no Opportunities for this Account, add it for deletion
                     if (thisAccount.Opportunities.size() == 0 && thisAccount.Contacts.size() == 0) {
```

Comment:
I'm not sure we need the Opportunities restriction on this either any longer. Typically it was done to prevent cascade deletion in scenarios where the contact still had donations, but in reality, there's enough other dependent information, that I'm thinknig we might need to just add another issue to determine what the correct cascade delete behavior we want to enable it